seppy/tools/__init__.py

```diff
@@ -431,11 +471,23 @@ def choose_data(self, viewing):
                     self.current_e_energies = self.energies_e_south
 
         if self.spacecraft.lower() == 'wind':
+
             if self.sensor.lower() == '3dp':
-                col_list_i = [col for col in self.df_i.columns if col.endswith(str(viewing))]
-                col_list_e = [col for col in self.df_e.columns if col.endswith(str(viewing))]
-                self.current_df_i = self.df_i[col_list_i]
-                self.current_df_e = self.df_e[col_list_e]
+            # The sectored data has a little different column names
+                if self.viewing == "omnidirectional":
+
+                    col_list_i = [col for col in self.df_omni_i.columns if "FLUX" in col]
+                    col_list_e = [col for col in self.df_omni_e.columns if "FLUX" in col]
+                    self.current_df_i = self.df_omni_i[col_list_i]
+                    self.current_df_e = self.df_omni_e[col_list_e]
+
+                else:
+
+                    col_list_i = [col for col in self.df_i.columns if col.endswith(str(self.viewing)) and "FLUX" in col]
+                    col_list_e = [col for col in self.df_e.columns if col.endswith(str(self.viewing)) and "FLUX" in col]
+                    self.current_df_i = self.df_i[col_list_i]
+                    self.current_df_e = self.df_e[col_list_e]
```

Comment:
Just wondering, is the `and "FLUX" in col` part actually necessary?

Reply:
![Screenshot from 2022-11-03 11-43-12](https://user-images.githubusercontent.com/50318651/199689552-b715864e-9f3c-48fd-bcac-c69f39bfd68a.png)
Yes it is, otherwise the dataframe will be two times bigger in size, and parsing channel indices in class methods down the road would be more difficult.